plotly/offline/offline.py

```diff
@@ -454,7 +454,6 @@ def plot(figure_or_data, show_link=True, link_text='Export to plot.ly',
         resize_script = (
             ''
             '<script type="text/javascript">'
-            'window.removeEventListener("resize");'
```

Comment:
just for some historical context:

I'm pretty sure that this line was in here for `offline.iplot`: when you would re-run `iplot` over and over, we wanted to remove the previously added `eventListener` that might've been added on the first `iplot` run.

From refactors of `plotly/offline/offline.py`, I think that this got copied and pasted around and is now in `plot` even though it doesn't need to be. `plot` doesn't need to call `removeEventListener` because the following `window.addEventListener` will only ever be added once in the `plot.html` context.